### ISSUE TITLE
ARIA attribute not being added correctly through accessibilityState

### ISSUE THREAD

**Issue by zero298** [necolas/react-native-web#1483]
**The problem**

Setting the `accessibilityState` on a `<TouchableOpacity/>` doesn't seem to be setting the appropriate aria attributes.

**How to reproduce**

Simplified test case: [Test Case](https://codesandbox.io/s/issue-react-native-for-web-uge6x)

Steps to reproduce:
1. Attempt to set the `accessibilityState` of a `<TouchableOpacity/>` by providing an Object per RN docs
2. Open DOM inspector and inspect the touchable element
3. The element does not have the appropriate `aria-*` attributes

<img width="1039" alt="Screen Shot 2019-11-13 at 2 31 37 PM" src="https://user-images.githubusercontent.com/325543/68802542-b5169700-0623-11ea-8741-4798662bc6d5.png">
<img width="1005" alt="Screen Shot 2019-11-13 at 2 33 24 PM" src="https://user-images.githubusercontent.com/325543/68802543-b5169700-0623-11ea-9bab-c77497e29950.png">

**Expected behavior**

I would expect the correct `aria-*` to be added.  In this case, it should be `aria-selected="true"`.

**Environment (include versions). Did this work in previous versions?**

* React Native for Web (version): `0.11.7`
* React (version): `16.10.2`
* Browser: Firefox and Chrome at least

I believe the issue is how the state props are being pulled off of the object. `createDOMProps`

https://github.com/necolas/react-native-web/blob/4040151ee643a02534e72deb51eee76f001b9a5f/packages/react-native-web/src/modules/createDOMProps/index.js#L67

is expecting `accessibilityState` to come in as an `Array`

https://github.com/necolas/react-native-web/blob/4040151ee643a02534e72deb51eee76f001b9a5f/packages/react-native-web/src/modules/createDOMProps/index.js#L108-L112

The test seems to support that:

https://github.com/necolas/react-native-web/blob/4040151ee643a02534e72deb51eee76f001b9a5f/packages/react-native-web/src/modules/createDOMProps/__tests__/index-test.js#L161-L166

However, React Native wants this to come in as [an Object](https://facebook.github.io/react-native/docs/accessibility#accessibilitystate-ios-android) like this:

```js

<TouchableOpacity
    accessibilityRole="tab"
    accessibilityState={{
        selected: true
    }}
>
    <Text>Some Text</Text>
</TouchableOpacity>
```

**Comment by necolas** [comment]
https://github.com/necolas/react-native-web/commit/d57fb6eb01f5226c01e034bd1ce535b5a7204b33

**Comment by oshimayoan** [comment]
@zero298 Did you get yours to work? I'm getting the same issue with the same version.

**Comment by zero298** [comment]
@oshimayoan I haven't tried to reproduce or check since. Once I do, I'll let you know.
